contrib/kube-prometheus/Makefile

```diff
@@ -51,6 +52,7 @@ test-in-docker: ../../hack/jsonnet-docker-image
 	--rm \
 	-u=$(shell id -u $(USER)):$(shell id -g $(USER)) \
 	-v $(shell dirname $(dir $(abspath $(dir $$PWD)))):/go/src/github.com/coreos/prometheus-operator/ \
+	-v $(shell go env GOCACHE):/.cache/go-build \
```

Comment:
As far as I know, go cache invalidation only happens based on golang code changes. If we change the environment itself, tests would not be rerun.

Thereby I find this a bit risky compared to the time saved through it. What do you think?

Reply:
Thank you for the comment, I agree as I had the same doubts, in the end I added the cache to the containers because go 1.12 requires the build cache.

Would invalidating the test cache solve this issue?
AFAIK to disable test caching you only need the `-count=1` flag:
https://golang.org/cmd/go/#hdr-Testing_flags
```
When 'go test' runs in package list mode, 'go test' caches successful package test results to avoid unnecessary repeated running of tests. To disable test caching, use any test flag or argument other than the cacheable flags. The idiomatic way to disable test caching explicitly is to use -count=1. 
```